modules/ROOT/pages/_partials/sync/server-realtime-updates.adoc

```diff
@@ -1,3 +1,125 @@
 = Realtime Updates
 
-TODO
\ No newline at end of file
+A core concept of GraphQL is subscriptions. These subscriptions can be leveraged to provide a mechanism for real time updates.
+To enable this on the server, an appropriate schema should be defined. A recommended implementation pattern can be seen in the schema below.
+
+```
+const { PubSub } = require('apollo-server');
```

Comment:
This is too verbose and as reader I have no clue what I should be focused on.
We have now nice separation in the showcase server. 
From the docs I get that there is some feature offered by voyager but we use apollo server. If we make that clear it will be easier for reader to get more info about subscriptions in google.